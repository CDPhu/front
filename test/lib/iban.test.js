const iban = require("../../lib/iban")

describe("iban.isValidIBANNumber", () => {
    test("0", () => {
        let result = iban.isValidIBANNumber("George")
        expect(result).toMatchSnapshot()
    })

    test("1", () => {
        let result = iban.isValidIBANNumber("Jean-Philippe")
        expect(result).toMatchSnapshot()
    })

    test("2", () => {
        let result = iban.isValidIBANNumber("Anas")
        expect(result).toMatchSnapshot()
    })

    test("3", () => {
        let result = iban.isValidIBANNumber("Edmond")
        expect(result).toMatchSnapshot()
    })

    test("4", () => {
        let result = iban.isValidIBANNumber("Pierre Edouard")
        expect(result).toMatchSnapshot()
    })

    test("5", () => {
        let result = iban.isValidIBANNumber("")
        expect(result).toMatchSnapshot()
    })
})
