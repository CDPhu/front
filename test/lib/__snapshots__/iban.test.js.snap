// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`iban.isValidIBANNumber 0 1`] = `false`;

exports[`iban.isValidIBANNumber 1 1`] = `false`;

exports[`iban.isValidIBANNumber 2 1`] = `false`;

exports[`iban.isValidIBANNumber 3 1`] = `false`;

exports[`iban.isValidIBANNumber 4 1`] = `false`;

exports[`iban.isValidIBANNumber 5 1`] = `false`;
